packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -236,6 +234,41 @@ abstract class MouseCursor with Diagnosticable {
   }
 }
 
+/// A special class that indicates that the region with this cursor defers the
+/// choice of cursor to the next region behind it.
+///
+/// When an event occurs, [MouseTracker] will update the each pointer's cursor by
+/// finding the list of regions that contain the pointer's location, from front
+/// to back in hit-test order. The pointer's cursor will be the first cursor in
+/// the list that is not a [DeferredMouseCursor].
+class DeferredMouseCursor extends MouseCursor {
+  const DeferredMouseCursor._();
+
+  /// An instance of [DeferredMouseCursor].
+  ///
+  /// It is the global single instance of [DeferredMouseCursor].
+  static const DeferredMouseCursor instance = DeferredMouseCursor._();
+
+  @override
+  MouseCursorSession createSession(int device) {
+    assert(false, 'DeferredMouseCursor can not create a session');
+    throw UnimplementedError();
+  }
+
+  @override
+  String get debugDescription => '';
```

Comment:
Surprised that an empty string is the best way to describe this cursor?

Reply:
Since this seems to be causing too much confusion I'm changing how it works. Previously `toString` returns `$runtimeType($debugDescription)`, now it returns `debugDescription`. Implementations are changes accordingly.